### PR TITLE
[FIX] hr_homeworking: disply label of homeworking

### PR DESCRIPTION
**Pull request by shup-odoo**
Steps:
- Open Employee
- Select kanban view.
- The label for homeworking is not showing like home/office/unspecified.

Issue:
- The label for homeworking is not showing in kanban view.

Cause:
- The field for displaying the homeworking label is not defined.

Fix:
- Added the field for displaying of homeworking label

Task: 3520312

### CODE REVIEW

File: addons/hr_homeworking/views/hr_employee_views.xml [hunk 1]
```diff
@@ -85,12 +85,13 @@
                 </t>
                 <t t-set="location_map" t-value="{'home': 'fa-home', 'office': 'fa-building', 'other': 'fa-map-marker'}"/>
                 <t t-set="location" t-value="record.hr_icon_display.raw_value.split('_')[1]"/>
-                <t t-set="icon" t-value="location_map[location]"/>
                 <t t-if="location !== 'holiday'">
-                    <span class="fa fa-2x" t-attf-class="{{icon}} {{text_color}}" role="img"
+                    <t t-set="icon" t-value="location_map[location]"/>
+                    <span class="fa fa-2x" t-attf-class="{{icon}} {{text_color}} " role="img"
```
Comment (robcoekaerts) [inline addons/hr_homeworking/views/hr_employee_views.xml:90]: What are these changes for?

File: addons/hr_homeworking/views/hr_employee_views.xml [hunk 2]
```diff
@@ -85,12 +85,13 @@
...
                         t-att-aria-label="`Working from ${location}`" t-att-title="`Working from ${location}`"
                         t-att-name="`presence ${record.hr_presence_state.raw_value} at ${location}`"
                     />
+                    <t field="name_work_location_display"/>
```
[the remark this answers is not in the document]
Comment (shup-odoo) [reply addons/hr_homeworking/views/hr_employee_views.xml]: @robcoekaerts Done the changes.

File: addons/hr_homeworking/views/hr_employee_views.xml (hunk 1 above)
Comment (shup-odoo) [reply addons/hr_homeworking/views/hr_employee_views.xml]: only set icon if location isn't 'holiday'

File: addons/hr/views/hr_employee_public_views.xml [hunk 3]
```diff
@@ -150,7 +150,7 @@
                                     <li id="last_login"/>
                                     <li id="presence">
                                         <t t-if="record.show_hr_icon_display.raw_value">
-                                            <div class="o_employee_availability align-items-center">
+                                            <div class="o_employee_availability d-flex align-items-center">
```
[the remark this answers is not in the document]
Comment (shup-odoo) [reply addons/hr/views/hr_employee_public_views.xml]: Before using the d-flex. the `icon` and `text` not showing properly.

File: addons/hr_homeworking/views/hr_employee_views.xml (hunk 1 above)
Comment (ppr-odoo) [reply addons/hr_homeworking/views/hr_employee_views.xml]: > What are these changes for?

When the location is  a holiday, the icon is useless, so I have moved inside the if condition. is not good way ?

Comment (robcoekaerts) [reply addons/hr_homeworking/views/hr_employee_views.xml]: Ah ok now I see what it's about. Sure it's a little useless, but it does no harm and changing it messes up the commit history and makes it less clear what this commit is actually about. Also by only changing it in the private view and not the public one, this two views are inconsistent.

I would prefer this change not to be there.

Comment (ppr-odoo) [reply addons/hr_homeworking/views/hr_employee_views.xml]: > Ah ok now I see what it's about. Sure it's a little useless, but it does no harm and changing it messes up the commit history and makes it less clear what this commit is actually about. Also by only changing it in the private view and not the public one, this two views are inconsistent.
> 
> I would prefer this change not to be there.

Okay. i revert this changes thanks

**Comment by ppr-odoo** [comment]
Task cancelled 

@shup-odoo thanks for the work

**Pull request closed without merging**